R/files.R

```diff
@@ -0,0 +1,58 @@
+#' Checks if the value is a file
```

Comment:
The name of this is ambiguous. What you are checking here is not if something is a file. What you are checking here is if a path exists AND it refers to a file.  

A file, in terms of the language, is some entity that refers to an open file. This is a file:

> x <- file("ciccio", "w")
> class(x)
[1] "file"       "connection"
> 

You can do various I/O operations on it.

This ambiguity is not as far fetched as you might think, because you can absolutely envision we write a routine that accepts an open file, rather than a path, and reads or writes information to the file, and expose this routine to the user. So we must be able to be unambiguous that we want a _file_, not a _path_ to a file, in the error message.

Reply:
I am not sure how to call the routine.

is_path_existent maybe? is_path_present? maybe with is_file vs. is_dir arguments?
Not sure.